app/components/TopNav.vue.ts

```diff
@@ -38,6 +38,11 @@ export default class TopNav extends Vue {
   slideOpen = false;
 
   studioModeTooltip = 'Studio Mode';
+  settingsTooltip = 'Settings';
+  helpTooltip = 'Get Help';
+  logoutTooltip = 'Logout';
+  sunTooltip = 'Day mode';
+  moonTooltip = 'Night mode';
```

Comment:
these tooltips should actually be wrappedlike so: `$t('Settings')` so they get picked up by our translations service but otherwise looks good :+1:

Reply:
I'll add these, thanks!

Reply:
https://github.com/stream-labs/streamlabs-obs/pull/1152/commits/aab49766ef93ac122f9b429be24ae0614dbf9cf3